COMPARE.md

```diff
@@ -1,7 +1,7 @@
 # Comparison with other libraries
 
 ## [Ramda](http://ramdajs.com/)
-Ramda was a massive influence on Tinkoff Utils It borrowed the same functional programming approach, naming, and many other things. However, Tinkoff Utils doesn't provide placeholders, lenses and some other less popular stuff.
+Ramda was a huge inspiration for Tinkoff Utils! it has a lot of similarities with naming style, uses same functional programming approach and many other things, with luck of some less popular design ideas like `__` prefixes, `lens*` functions and etc
```

Comment:
Ahem

Reply:
Why did you revert my changes?

Reply:
Fixed

Reply:
Not reverted but augmented I think and not all of it - my goal was to make the language less formal.

Whereas that phrase –

> However, Tinkoff Utils doesn't provide placeholders

Might imply that this tool doesn't provide some additional features

Reply:
I was talking about this particular paragraph. It contains some original mistakes and typos, e.g., using "luck" instead of "lack".

Reply:
Fixed, anything else?